### ISSUE TITLE
if you update the number of bins, the old bins remains displayed as a gray shade

### ISSUE THREAD

**Issue by javisantana** [CartoDB/deep-insights.js#166]
cc @abelvm

**Comment by AbelVM** [comment]
Example:

Initially, bins=6, updated to 24 
```javascript
if (mywidget.attributes.bins != 24) mywidget.update({bins: 24});
```
and then:

![img](http://i.imgur.com/BC2sPmh.png)

**Comment by AbelVM** [comment]
The X-axis labels remains unchanged as well

**Comment by acanimal** [comment]
@AbelVM just for curiosity, why do yo need to change the number of bins?

Right now the histogram was though to have a set of bins, create it shadow bars and also allow to zoom on bins. Modifying to allow change bins dynamically requires make some big modifications of the component.

**Comment by AbelVM** [comment]
Because I can't set a fixed number of bins on di.json right now :disappointed:

**Comment by acanimal** [comment]
@AbelVM i close this issue because the problem to specify initial bin number will be solved in #164

**Comment by xavijam** [comment]
No, if you update the number of bins the histogram widget should be refreshed, shadow bars, green bars and labels.
